package/tests/test_api/test_graphql/test_queries.py

```diff
@@ -7,6 +7,21 @@
 
 from kedro_viz import __version__
 
+try:
+    from kedro_datasets import tracking, plotly, matplotlib  # isort:skip
+except ImportError:
+    from kedro.extras.datasets import (  # Safe since ImportErrors are suppressed within kedro.
+        matplotlib,
+        plotly,
+        tracking,
+    )
+
+
+def _get_dataset_type(dataset):
```

Comment:
It would be good to have a comment here about what this function does, and how and why it is different from the function with the same name in `package/kedro_viz/models/flowchart.py`, as this may get confusing.

Reply:
u r right. we don't need to call this here. we can jsut import it

Reply:
u right these two functions are not required in the test files (test_queries, and test _responses). in order to assert the correct result I am just going to use string "pandas.csv_dataset.CsvDataSet" just the way it was done before